components/Buttons/BasicButton/BasicButton.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export default function BasicButton({ text }) {
+  return <StyledButton>{text}</StyledButton>;
+}
```

Comment:
For this "basic" button without any fancy stuff, it would be enough to only export the styled component instead of this React Component.